.yarnrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
https://medium.com/@crysfel/using-different-registries-in-yarn-and-npm-766541d6f851